commit e396131465afa57ea2eaa7412aa72749013f2bb6
Author: Pawel Tomulik <ptomulik@meil.pw.edu.pl>
Date: 1462925850 -7200

    fixed issue with _xxxxxxVERSIONFLAGS

diff --git a/engine/SCons/Defaults.py b/engine/SCons/Defaults.py
--- a/engine/SCons/Defaults.py
+++ b/engine/SCons/Defaults.py
@@ -540,10 +540,10 @@
             frame = frame.f_back
         return None
 
-# if env[version_var] id defined, returns env[flags_var], otherwise returns None
+# if $version_var is not empty, returns env[flags_var], otherwise returns None
 def __libversionflags(env, version_var, flags_var):
     try:
-        if env[version_var]:
+        if env.subst('$'+version_var):
             return env[flags_var]
     except KeyError:
         pass

